VolunteerMeet-MoMs/MOM-23rd-Nov-2019

```diff
@@ -0,0 +1,11 @@
+1.  Team structure and Chair roles are assigned.
+2.  Co-Chair and assistant chair roles are assigned. (Some assistant chairs are empty as volunteers were not present on the day.)
```

Comment:
Change this with - Room Monitors and Session chairs.

Reply:
Also, please add this spreadsheet in the MoM - https://docs.google.com/spreadsheets/d/17q3E5F2c8cwTBLyHL2ly4Ik7UUIbs-D80KCSNysdvvM/edit#gid=0

Reply:
I've made the changes.